msb/rawlogger/msb_rawlogger.py

```diff
@@ -32,20 +32,24 @@ def get_data(self) -> tuple[bytes, bytes]:
                 continue
             yield topic, data
 
+
     def run(self):
         for topic, data_raw in self.get_data():
-            if self.config.print_stdout:
-                print(f"{topic.decode('utf-8')} : {data_raw.decode('utf-8')}")
+            topic_decoded = topic.decode('utf-8')
+            if topic_decoded not in self.excluded_topics:
+                print(f"{topic_decoded} : {data_raw.decode('utf-8')}")
```

Comment:
Now all data is printed regardless of `self.config.print_stdout`. Please fix.

Reply:
fixed this issue.